Core/Core/RichContentEditor/RichContentEditor.css

```diff
@@ -10,40 +15,36 @@ html, body {
 }
 #content {
     box-sizing: border-box;
-    min-height: 100%;
-    padding: 1em;
+    min-height: 1.25rem;
+    padding: 16px;
 }
 #content:empty::before,
 #content.show-placeholder::before {
     content: attr(placeholder);
-    color: var(--text-dark);
+    color: var(--color-textDark);
     position: absolute;
 }
-.is-hidden {
-    display: none;
-}
 svg {
     display: block;
 }
 [data-media_comment_id] {
-    background-color: var(--background-darkest);
+    background-color: var(--color-backgroundDarkest);
     width: 288px;
 }
 [data-uploading] {
     opacity: 0.25;
 }
 .image-overlay {
```

Comment:
Most of the following css changes were to make the image uploading stuff layout correctly in RTL.